app/protocol/flows/v4/blockrelay/ibd_with_headers_proof.go

```diff
@@ -24,6 +24,7 @@ func (flow *handleIBDFlow) ibdWithHeadersProof(highHash *externalapi.DomainHash,
 			return err
 		}
 
+		log.Infof("IBD with pruning proof from %s was unsuccessful. Deleting the staging consensus.", flow.peer)
```

Comment:
Right after this, the user sees the line `Deleting database prefix &{0}` which seems worrying. We want to explain first.